static/app/components/group/tagFacets/tagFacetsDistributionMeter.tsx

```diff
@@ -4,6 +4,7 @@ import styled from '@emotion/styled';
 import debounce from 'lodash/debounce';
```

Comment:
segmentProps for <SegmentBar> in this file could also use aria-labels (I see we have it for "Other" but not the other segments)
<img width="619" alt="Screen Shot 2023-01-25 at 3 54 40 PM" src="https://user-images.githubusercontent.com/63818634/214688408-f4263636-9544-4c5e-b342-e0e0e8cedde1.png">

Reply:
Added labels for segments!